contracts/SpokePool.sol

```diff
@@ -162,56 +139,11 @@ abstract contract SpokePool is
     event SetXDomainAdmin(address indexed newAdmin);
     event SetHubPool(address indexed newHubPool);
     event EnabledDepositRoute(address indexed originToken, uint256 indexed destinationChainId, bool enabled);
-    event RequestedSpeedUpDeposit(
```

Comment:
I'm wondering if there's value in retaining these event definitions somewhere, so we can continue to reference them in scripts that look back beyond the upgrade bump. Otherwise we'll to define those separately somewhere else, or handle multiple contracts-v2 versions somehow. wdyt?

Reply:
Isn't the git history good enough to serve as ground truth? Additionally why not retain these in legacy files in the SDK if we want to query them for some reason. 

I don't love polluting the contracts repo with deprecated code but I'm open to suggestions

Reply:
Yeah to be clear, I wasn't suggesting to retain it in this file, but maybe somewhere else in this repo so we continue to get the benefit of the existing tooling (i.e. typechain). I'm not sure what the best way to handle deprecated contract functionality is :thinking:.